tstest/integration/integration_test.go

```diff
@@ -230,6 +230,7 @@ func TestNodeAddressIPFields(t *testing.T) {
 }
 
 func TestAddPingRequest(t *testing.T) {
+	t.Skip() // TODO: https://github.com/tailscale/tailscale/issues/2079
```

Comment:
LGTM. Usually you put the reason in the Skip message, though:

<img width="1051" alt="Screen Shot 2021-06-14 at 7 42 08 PM" src="https://user-images.githubusercontent.com/2621/121984883-9c62eb80-cd48-11eb-9865-8f1cbdf73073.png">

Reply:
Done.